0.6.0

gilbarbara opened the pull request:
- update `changed` to accept `actual` and `previous`
- deprecate `changedTo` in favor of the updated `changed`
- add `added`, `emptied`, `filled` and `removed

carlosasj commented on src/base.ts:
```diff
@@ -0,0 +1,183 @@
+    const right = nested(data, key);
+    const hasActual = isDefined(actual);
+    const leftComparator = isArray(previous) ? previous.indexOf(left) >= 0 : left === previous;
+    const rightComparator = isArray(actual) ? actual.indexOf(right) >= 0 : right === actual;
```
this pattern. You could modularize it:
```js
function includesOrEqualsTo<T>(expectedValue: T | T[], value: T): boolean {
  return isArray(expectedValue) ? expectedValue.includes(value) : value === expectedValue;
}
```
and use as
```js
const leftComparator = includesOrEqualsTo(previous, left);
const rightComparator = includesOrEqualsTo(actual, right);
```

gilbarbara committed:
Upgrade dependencies

- add esm build
- replace tslint with eslint
- replace deep-diff with fast-deep-equal
- remove nested-property

gilbarbara committed:
Update API

- update `changed` to accept `actual` and `previous`
- deprecate `changedTo` in favor of the updated `changed`
- add `added`, `emptied`, `filled` and `removed` methods

gilbarbara committed:
Add react hook

gilbarbara committed:
Add demo
